package.json

```diff
@@ -60,7 +61,8 @@
     "solidity-coverage": "^0.5.11",
     "solium": "^1.1.8",
     "truffle": "5.0.0-next.17",
-    "truffle-keystore-provider": "^1.0.2",
-    "truffle-assertions": "0.6.3"
-  }
+    "truffle-assertions": "0.6.3",
+    "truffle-keystore-provider": "^1.0.2"
+  },
+  "dependencies": {}
```

Comment:
Ah no probably not I first installed `abi-decoder` in deps rather than devDeps , then uninstalled it which is why there's now an empty object there. 

Since we're using `truffleAssert.createTransactionResult()` we no longer need `abi-decoder` either, meaning we can completely revert all changes to `package.json`

Reply:
fixed in ad3cb40